test/expanded-comp/locales/expanded-comp__fr.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@redbugz 
I did intend to use 2 underscores here. I was going to test if having a `something_fr.json` *and* a `something__fr.json` would be a problem but maybe we don't need to worry about this kind of case?